Automattic/jetpack
# Publicize Components: Add the new package

#### Changes proposed in this Pull Request:

This PR adds the new Publicize Components JS package, and moves the
store, connection and twitter components to it. It includes a minor edit
to the twitter options component, to satisfy a lint rule.

#### Jetpack product discussion
pdrWKz-cd-p2

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
Check the unit tests with `jetpack test js-packages/publicize-components js`

With this PR, check that the Publicize panel still works as before. Some things to check:

- Toggling the 'Share this post' off should prevent the post being publicized
- Similarly toggling off any connection should prevent the post being shared through that connection
- Changing it to a twitter thread, should also work, and you'll still see the tweet divider on the canvas
